packages/angular/src/app/modules/builder/components/builder-component/builder-component.component.ts

```diff
@@ -149,7 +149,6 @@ export class BuilderComponentComponent implements OnDestroy, OnInit {
         builder.apiKey = this.builderService.apiKey;
         builder.canTrack = this.builderService.canTrack;
         builder.setUserAttributes(omit(this.builderService.getUserAttributes(), 'urlPath'));
-        builder.userAttributesChanged.subscribe((attrs) => builder.setUserAttributes(attrs));
```

Comment:
oops, this is supposed to be here, but should be

```ts
        this.builderService.userAttributesChanged.subscribe((attrs) => builder.setUserAttributes(attrs));	
```

Reply:
that did fix the exception